src/ImportInfoDialog.cpp

```diff
@@ -0,0 +1,39 @@
+#include "ImportInfoDialog.h"
+
+BEGIN_EVENT_TABLE( ImportInfoDialog, wxDialogWrapper )
+   EVT_BUTTON( wxID_CANCEL, ImportInfoDialog::OnCancel )
+END_EVENT_TABLE()
+
+ImportInfoDialog::ImportInfoDialog(const FilePath& filename, wxWindow* parent, wxWindowID id, const TranslatableString &title, const wxPoint &position, const wxSize& size, long style ):
+      wxDialogWrapper( parent, id, title, position, size, style )
+{
+    wxBoxSizer* sizer = new wxBoxSizer(wxVERTICAL);
+
+    m_statusText = new wxStaticText(this, wxID_ANY, wxString::Format(wxT("Importing %s..."), filename.AfterLast(wxFileName::GetPathSeparator())));
+    sizer->Add(m_statusText, 1, wxALL | wxEXPAND, 10);
+
+    wxButton* cancelButton = new wxButton(this, wxID_CANCEL, wxT("Cancel"));
+    sizer->Add(cancelButton, 0, wxALL | wxCENTER, 10);
+
+    SetSizerAndFit(sizer);
+    
+    int parentWidth = -1, parentHeight=-1;
+    if( GetParent() )  GetParent()->GetSize( &parentWidth, &parentHeight );
+    if (parentWidth > 400)
+       CenterOnParent();
+    else
+       CenterOnScreen();
+
+    Bind(wxEVT_BUTTON, &ImportInfoDialog::OnCancel, this, wxID_CANCEL);
```

Comment:
Handler is already bound to the event via event table you've defined above, one of them should be removed

Reply:
fixed